src/backends/torch/torchlib.cc

```diff
@@ -248,6 +248,10 @@ namespace dd
       {
         _seq_training = true;
       }
+    else if (_template == "fasterrcnn" || _template == "retinanet")
+      {
+        // bbox = true; // ?
```

Comment:
We could force it automatically yes.

Reply:
Do we need to pass it to the output parameters at predict time then?

Reply:
Normally yes, but if we hardcode it, it'll provide some slack. There's no use not setting it up actually.